apps/member-profile/app/shared/components/review-form.tsx

```diff
@@ -225,6 +235,18 @@ function RecommendField({ defaultValue, error, name }: FieldProps<boolean>) {
   );
 }
 
+function AnonymousField({ error, name, defaultValue }: FieldProps<boolean>) {
+  return (
+    <Form.Field
+      error={error}
+      label="Would you like to remain anonymous?"
```

Comment:
nit: we could change this to `Would you like to post this review anonymously?`

Reply:
fixed in `49cf515aa4c266e18984609d1c5a04b01488dc5a`